### ISSUE TITLE
Feature Request: Change order of elements in the top overview

### ISSUE THREAD

**Issue by ErikKoppenhagen** [OpenKieler/osgiviz#3]
In the top overview, the elements "_products_", "_features_", "_bundle categories_", "_bundles_" and "_services_" are shown, but not in that order.
It would be nice to some some kind of "_contains_" logic in their arrangement for a better overview.

**Comment by NiklasRentzCAU** [comment]
Yes, I can see that would be nice. I experimented with the new rectangle packing algorithm of ELK to support sorting of these elements while still having a nice layout of those overviews (not just everything stacked on top of each other), there are however two issues that have to be fixed before I will use that algorithm here.
See [here](https://github.com/eclipse/elk/issues/482) and [here](https://github.com/eclipse/elk/issues/483).
